src/components/views/dialogs/BaseDialog.js

```diff
@@ -56,9 +56,8 @@ export default React.createClass({
         // CSS class to apply to dialog div
         className: PropTypes.string,
 
-        // Title for the dialog.
-        // (could probably actually be something more complicated than a string if desired)
-        title: PropTypes.string.isRequired,
+        // Title for the dialog. Can be a string or node.
+        title: PropTypes.any.isRequired,
```

Comment:
'node' is what you want here (a node is anything that can be rendered, eg string / react element. 'element' is just a react element).